src/composables/createMultisig.ts

```diff
@@ -0,0 +1,85 @@
+import { ref } from '@vue/composition-api';
+import type { Store } from 'vuex';
+import multisigContract from '@aeternity/ga-multisig-contract/contracts/SimpleGAMultiSig.aes';
+import { Crypto } from '@aeternity/aepp-sdk';
+import { MultisigAccount, MultisigCreationStep } from '../types';
+import { useSdk } from './sdk';
+
+interface UseMultisigOptions {
+  /**
+   * TODO: Temporary solution to avoid dependency circle
+   */
+  store: Store<any>
+}
+
+const MINIMUM_REQUIRED_SIGNERS = 2;
+
+/**
+ * This composable provides methods related to multisig operations
+ */
+export function useMultisig({ store }: UseMultisigOptions) {
+  const { getDrySdk, getSdk } = useSdk({ store });
+
+  const multisigProgress = ref<MultisigCreationStep>('compiled');
```

Comment:
I think the default value should be 'preparing'
```suggestion
  const multisigProgress = ref<MultisigCreationStep>('preparing');
```